config/webpack.config.js

```diff
@@ -27,5 +29,25 @@ module.exports = {
   },
   resolve: {
     extensions: ['.ts', '.js', '.tsx', '.jsx'],
+    fallback: {
```

Comment:
Had a few issues with `@walletconnect/web3-provider`, needed this "polyfills" to work

Reply:
Related https://github.com/coinbase/coinbase-wallet-sdk/issues/56

Reply:
Related https://github.com/WalletConnect/walletconnect-monorepo/issues/748

Reply:
Could use https://github.com/Richienb/node-polyfill-webpack-plugin instead